source/boot/boot.js

```diff
@@ -7,19 +7,18 @@
 		sheet: function (inPath) {
 			var type = 'text/css';
 			var rel = 'stylesheet';
-			var isLess = (inPath.slice(-5) == '.less');
-			if (isLess) {
-				if (window.less) {
-					// If client-side less is loaded, insert the less stylesheet
-					type = 'text/less';
-					rel = 'stylesheet/less';
-				} else {
-					// Otherwise, we expect a css file of the same name to exist
-					inPath = inPath.slice(0, inPath.length-4) + 'css';
-				}
+			var isCss = (inPath.slice(-4) == '.css');
+			var isLess = !isCss || (inPath.slice(-5) == '.less');
+			if (window.less && (isLess || isCss)) {
+				// If client-side less is loaded, insert the less stylesheet
+				type = 'text/less';
+				rel = 'stylesheet/less';
+			} else if (isLess) {
+				// Otherwise, we expect a css file of the same name to exist
+				inPath = inPath.slice(0, inPath.length-4) + 'css';
```

Comment:
We should also consider turning on debugging when run from the debug.html file. So when errors DO happen, they're apparent. I wrote a stylesheet for an app that makes the debug output readable when used with a moonstone app which we can bake into enyo for consistency.